packages/components/tag/src/tag-list.ts

```diff
@@ -113,24 +113,14 @@ export class TagList extends ScopedElementsMixin(LitElement) {
     if (changes.has('stacked') && !this.stacked) {
       this.tags.forEach(tag => (tag.style.display = ''));
     }
-
-    if (changes.has('stacked') || changes.has('stackSize') || changes.has('tags')) {
-      if (this.stacked) {
-        const total = this.tags.length;
-
-        this.setAttribute('aria-label', `${msg(str`Showing ${total - this.stackSize} out of ${total} items`)}`);
```

Comment:
Why are we removing this part? This part was added before to improve the accessibility and inform the user how many items are currently visible in the stacked version.

Reply:
After testing with VoiceOver, the `role="list"` on the `<sl-tag-list>` already causes it to announce how many items are in the list. And with the `aria-labelledby` instead of `aria-describedby` on the stacked tag, it now also announces which tags are in the stack. 
And finally, this aria-label blocks any aria-labels set from the outside of the tag-list, like `aria-label="Selected options"` from the combobox.